src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -160,75 +105,98 @@ export default function ToeristischeVerhuur() {
-            die dit jaar door een mede-verhuurder of vorige bewoner zijn gedaan.
-            Kijk voor meer informatie bij{' '}
-            <LinkdInline
+        {actualVergunningen?.length && (
+          <p>
+            <Linkd
+              external={true}
+              href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/vakantieverhuur/"
+            >
+              Meer informatie over regels voor Particuliere vakantieverhuur
+            </Linkd>
+            <br />
+            <Linkd
+              external={true}
+              href="https://www.amsterdam.nl/veelgevraagd/?productid=%7BF5FE8785-9B65-443F-9AA7-FD814372C7C2%7D"
+            >
+              Meer over toeristenbelasting
+            </Linkd>
+          </p>
+        )}
+        {!actualVergunningen?.length && BBActualVergunningen?.length && (
+          <p>
+            <Linkd
               external={true}
-              href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/vakantieverhuur/melden"
+              href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/bedandbreakfast/"
             >
-              Vakantieverhuur melden
-            </LinkdInline>
-            . Aan de informatie op deze pagina kunnen geen rechten worden
-            ontleend.
+              Meer informatie over bed and breakfast
+            </Linkd>
+            <br />
+            <Linkd
+              external={true}
+              href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/bedandbreakfast/regels/"
+            >
+              Regels bed & breakfast
+            </Linkd>
+          </p>
+        )}
+        <div className={styles.Detail}>
+          {!!actualVergunningen?.length && (
+            <>
+              <Heading el="h3" size="tiny" className={styles.InvertedLabel}>
+                U heeft nog {content?.daysLeft ?? 30} dagen dat u uw woning mag
+                verhuren.
+              </Heading>
+              <p className={styles.DetailText}>
+                Het aantal resterende nachten is gebaseerd op uw meldingen voor
+                ingepland en afgelopen verhuur. Dit is zonder eventuele
+                meldingen die dit jaar door een mede-verhuurder of vorige
+                bewoner zijn gedaan. Kijk voor meer informatie bij{' '}
+                <LinkdInline
+                  external={true}
+                  href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/vakantieverhuur/melden"
+                >
+                  Vakantieverhuur melden
+                </LinkdInline>
+                . Aan de informatie op deze pagina kunnen geen rechten worden
+                ontleend.
+              </p>
+            </>
+          )}
+          <p>
+            {!!actualVergunningen?.length && (
+              <>
+                {actualVergunningen.map((vergunning) => (
+                  <>
+                    <Linkd external={true} href={vergunning.link.to}>
```

Comment:
Dit zijn geen externe links. Wanneer de link **niet extern** is wordt er een react-router link gebruikt.